person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(name, age = 'unknown', parent_permission = 'true')
+    @name = name
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure you update all the parameter's default values to match the ones given in **[the project;s requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md)**.  `parent_permission` should be `boolean` and not string, `name` should have `"Unknown"` as the default value,...
```ruby
# Bad
  def initialize(name, age = 'unknown', parent_permission = 'true')

# Good
  def initialize(age, name = 'Unknown', parent_permission: true)
```